### PR TITLE
Stage 071 update

### PR DESCRIPTION
**Pull request by Evildoor**
Remove crutch per https://trello.com/c/WtXqpcrJ and implement updates to README and description that have been forgotten when the corresponding updates to code were made.

### CODE REVIEW

**Comment by Evildoor** [comment]
Added [WIP] until problems reported by travis are resolved.

**Comment by Evildoor** [comment]
Replaced [WIP] with [pending] as it was established that problem is in travis, and not in the PR.

File: Utils/Dataflow/071_esConsistency/README [hunk 1]
```diff
@@ -28,4 +28,3 @@ The stage can be run as following:
 For more information about running the stage and its arguments, use:
 
   ./consistency.py -h
-
```
Comment (mgolosova) [inline Utils/Dataflow/071_esConsistency/README]: The commit log states: `Remove second trailing empty line.`
Why is it "second"?

**Comment by Evildoor** [comment]
> https://github.com/PanDAWMS/dkb/blob/550f84ada91bd95e67506446ab30adb58dbe6d52/Utils/Dataflow/071_esConsistency/README#L15-L20
> 
> Maybe it would be better to first introduce the term "service fields", and then explain things like which service fields are required and how they are used?

True. Done.

File: Utils/Dataflow/071_esConsistency/README (hunk 1 above)
Comment (Evildoor) [reply Utils/Dataflow/071_esConsistency/README]: Changed "second" to "extra".

**Comment by mgolosova** [comment]
[Stage 071: убрать "костыли", относящиеся к лишним полям](https://trello.com/c/WtXqpcrJ/101-stage-071-%D1%83%D0%B1%D1%80%D0%B0%D1%82%D1%8C-%D0%BA%D0%BE%D1%81%D1%82%D1%8B%D0%BB%D0%B8-%D0%BE%D1%82%D0%BD%D0%BE%D1%81%D1%8F%D1%89%D0%B8%D0%B5%D1%81%D1%8F-%D0%BA-%D0%BB%D0%B8%D1%88%D0%BD%D0%B8%D0%BC-%D0%BF%D0%BE%D0%BB%D1%8F%D0%BC)

**Pull request merged**